docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -50,6 +50,29 @@ If your issue has not been resolved by this procedure, perform the following ste
 . If no existing issue addresses the encountered issue, link:https://github.com/code-ready/crc/issues/new[create an issue] and link:https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests[attach the [filename]`~/.crc/crc.log` file] to the created issue.
 The [filename]`~/.crc/crc.log` file contains detailed debugging and troubleshooting information which can help diagnose the problem that you are experiencing.
 
+= Getting shell access to the OpenShift cluster
+
+One of the core premises of OpenShift 4 (which is based on Red Hat CoreOS) is that one should not need to make direct changes to the cluster nodes.
+Instead, OpenShift operators will be doing all the needed changes on the nodes when you deploy a service or an application.
+Direct access to the OpenShift cluster is thus not needed in regular use, and strongly discouraged.
+If such access is needed for troubleshooting or debugging purposes, follow these steps:
+
+  . Enable `oc` access to the cluster, logging in as `kubeadmin`. For detailed steps, see <<accessing-the-openshift-cluster_{context}>>.
+
+  . Run [command]`oc get nodes`. The output will be similar to this:
++
+[subs="+quotes,attributes",options="nowrap"]
+----
+$ oc get nodes
+NAME                 STATUS   ROLES           AGE    VERSION
+crc-shdl4-master-0   Ready    master,worker   7d7h   v1.14.6+7e13ab9a7
+----
++
+
+  . Run [command]`oc debug nodes/crc-shdl4-master-0` where `crc-shdl4-master-0` is the name of the node which was printed at the previous step.
```

Comment:
I wonder if we'd be better off supplying a generic name for the node here rather than an explicit one which is inferred to be replaceable. Does that make sense?

Reply:
Nope, I'm not sure what you mean :(
Use `<node>` for example instead of `crc-shdl4-master-0` and explain that the value of <node> comes from the previous command? Or do you mean something else?

Reply:
I've pushed a new iteration with all the comments addressed except for this one, I'll push another series once what you expect is clearer to me :)

Reply:
Honestly, we're probably fine with this as it's written. I just wonder whether it's a good idea or not to use the node name explicitly, but this may be clearer as-is than using a generic name would be. :)